themes/2015/mix-manifest.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Should not be here, but in static/assets, something weird happened... (flo)

Reply:
Thanks I'll fix it.